slog/slog.go

```diff
@@ -51,6 +54,17 @@ type Config struct {
 	Format Format
 }
 
+// Fatal is equivalent to [Logger.Error] followed by a call to os.Exit(1).
+func (l *Logger) Fatal(msg string, args ...any) {
+	l.Error(msg, args...)
+	os.Exit(1)
+}
+
+// With calls Logger.With on the default logger returning a new Logger instance.
+func (l *Logger) With(args ...any) *Logger {
```

Comment:
We need this so chaining with With won't return a different type (Go's slog.Logger).